.github/workflow/test.yml

```diff
@@ -0,0 +1,40 @@
+name: continuous-integration
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - master
```

Comment:
@Harveyt47 this should be activated on pull-requests in general and not restricted to `master`

Reply:
`Github Actions` should show up once you make this change

Reply:
Hi @mmcky I've done this and I don't see `GitHub Actions` showing up